src/components/atom/button/index.ts

```diff
@@ -1,2 +1,2 @@
-import Button from "./button"
+import Button from "./Button"
```

Comment:
@Maurice63 not sure why this keeps showing, but can you exclude this file on your next commits unless there are actual changes applied? Only commit the files that you have intended changes. Thanks!

Reply:
I'm also not sure about that Sir, typeScript keeps on checking that on yarn run start but I'll keep that in mind.